cty/value_ops.go

```diff
@@ -74,9 +74,13 @@ func (val Value) GoString() string {
 // as dynamic if either of the given values are. Use RawEquals to compare
 // if two values are equal *ignoring* the short-circuit rules.
 func (val Value) Equals(other Value) Value {
+	// Nulls are equal, regardless of type.
```

Comment:
While we're here, I notice the doc comment above is lagging behind reality a little here, even before this change. Would you mind making the following revision while we're here?

```go
// Equals returns True if the receiver and the given other value have the
// same type and are exactly equal in value.
//
// As a special case, two null values are always equal regardless of type.
//
// The usual short-circuit rules apply, so the result will be unknown if
// either of the given values are.
//
// Use RawEquals to compare if two values are equal *ignoring* the
// short-circuit rules and the exception for null values.
```